chromadb/api/fastapi.py

```diff
@@ -98,8 +98,11 @@ def _modify(self, current_name: str, new_name: str, new_metadata: Optional[Dict]
 
     def delete_collection(self, name: str):
         """Deletes a collection"""
-        resp = requests.delete(self._api_url + "/collections/" + name)
-        resp.raise_for_status()
+        try:
+            resp = requests.delete(self._api_url + "/collections/" + name)
+            resp.raise_for_status()
+        except requests.exceptions.HTTPError:
+            raise ValueError(f"Collection with name {name} does not exist")
```

Comment:
I don't think we need this because we have middleware to send the actual error through, https://github.com/chroma-core/chroma/blob/main/chromadb/server/fastapi/__init__.py#L60, 

I will test it though :)